runhouse/resources/hardware/on_demand_cluster.py

```diff
@@ -197,6 +197,15 @@ def image_id(self) -> str:
             return self.image.image_id
         return None
 
+    @property
+    def creds_values(self) -> Dict:
+        if self._creds is None:
+            if self.launcher == LauncherType.DEN:
+                self._creds = DenLauncher.load_creds()
+            elif self.launcher == LauncherType.LOCAL:
+                self._creds = LocalLauncher.load_creds()
+        return self._creds.values
```

Comment:
we did `return {**self._creds.values, **self.ssh_properties}` for consistency with prior behavior. I think it's fine to change it to just return creds values (since creds arent shared but ssh properties are), but we should update `creds_values` for static clusters to match this behavior too